hack/testing/check-EFK-running.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
```

Comment:
It's good form in bash scripts to output the commands as they're run and to fail if any commands fail (avoid silent bugs).

    set -ex

http://www.peterbe.com/plog/set-ex

Requires a little more rigor but can save a lot of headaches.